src/test/java/org/jitsi/meet/test/ConferenceMigrationTest.java

```diff
@@ -94,10 +94,12 @@ public void testConferenceMigration()
             "Migrated bridge: " + migratedBridge +
             ", REST endpoint: " + jvbRESTEndpoint);
 
-        ensureOneParticipant("config.enforcedBridge=\"" + migratedBridge +"\"");
-        Participant owner = getParticipant1();
+        ensureTwoParticipants(
```

Comment:
Because of "ensureTwo/ThreeParticipants" methods we have so many troubles with overloads. Once will have only URL, the other time it will also have participant options. What if we start adding "ensureFour/Five/SixParticipants" ? I think we would have less problems and the code would be simpler if we'd change those to:
`
Participant p1 = createParticipant(0 /* index/number */, participantOptions);
Participant p2 = createParticipant(1, participantOptions);
p1.join(url);
p2.join(url);
`
To make porting easier we could leave "ensureOne/Two/Three", but only if there are no arguments. WDYT ?

Reply:
Also we should think of a proper way for dealing with "joinRef". We could have new "web participant option" which tells WebParticipant that it should use another way to join the conference.